integration/contractdeployer/contract_deployer_test.go

```diff
@@ -154,12 +162,12 @@ func createObscuroNetwork(t *testing.T) {
 }
 
 // Returns a viewing-key client with a registered viewing key.
-func getClient(wallet wallet.Wallet) *rpc.EncRPCClient {
+func getClient(hostWSPort int, wallet wallet.Wallet) *rpc.EncRPCClient {
 	viewingKey, err := rpc.GenerateAndSignViewingKey(wallet)
 	if err != nil {
 		panic(err)
 	}
-	client, err := rpc.NewEncNetworkClient(nodeAddress, viewingKey, testlog.Logger())
+	client, err := rpc.NewEncNetworkClient(fmt.Sprintf("ws://%s:%d", "127.0.0.1", hostWSPort), viewingKey, testlog.Logger())
```

Comment:
There's a network.Localhost constant used earlier in this file that you can use instead of the IP string